com.rclone.manager.yml

```diff
@@ -0,0 +1,80 @@
+id: com.rclone.manager
+
+runtime: org.gnome.Platform
+runtime-version: '49'
+sdk: org.gnome.Sdk
+
+command: rclone-manager
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications # When UI not openned, it uses notifications to show status. --talk-name=org.freedesktop.portal.Desktop not working very well.
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.secrets # For storing encrypted config passwords
+  - --system-talk-name=org.freedesktop.NetworkManager # For detecting is metered or not/
+  - --talk-name=org.freedesktop.Flatpak # Required for flatpak-spawn to access host commands
+  - --talk-name=org.freedesktop.portal.Desktop
+  - --filesystem=xdg-run/tray-icon:create
+  - --filesystem=home # For accessing user rclone config and other files
+modules:
+  - name: fusermount-wrapper # Need for mount and other rclone operations
+    buildsystem: simple
+    sources:
+      - type: file
+        path: fusermount-wrapper.sh
+    build-commands:
+      - install -Dm755 fusermount-wrapper.sh /app/bin/fusermount3
+
+  - name: lsof-wrapper # Need for track rclone
+    buildsystem: simple
+    sources:
+      - type: file
+        path: lsof-wrapper.sh
+    build-commands:
+      - install -Dm755 lsof-wrapper.sh /app/bin/lsof
+
+  - name: terminal-wrapper # Need for opening terminals from within flatpak
+    buildsystem: simple
+    sources:
+      - type: file
+        path: terminal-wrapper.sh
+    build-commands:
+      - install -Dm755 terminal-wrapper.sh /app/bin/kgx
+      - install -Dm755 terminal-wrapper.sh /app/bin/gnome-terminal
+      - install -Dm755 terminal-wrapper.sh /app/bin/xterm
+      - install -Dm755 terminal-wrapper.sh /app/bin/konsole
+      - install -Dm755 terminal-wrapper.sh /app/bin/alacritty
+
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: prebuilt-binaries # We already have builded apps so we don't need to build again.
```

Comment:
1. the name is too generic.
2. This should be built from source.

https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source